openapi.yaml

```diff
@@ -752,8 +803,12 @@ paths:
                       Conditions de correspondance à sélectionner, demandes de publication de données `application/json` uniquement 
 
                   examples:
-                    (Value not supported for GET requests):
-                      value: ''
+                    en:
+                      (Value not supported for GET requests):
+                        value: ''
+                    fr:
+                      (Valeur non supportée pour les requêtes OGE):
```

Comment:
is "OGE" confusing here because it doesn't match the actual HTTP verb?

Reply:
Good call, changed it back to GET